src/lib/readmeAPIFetch.ts

```diff
@@ -182,15 +179,17 @@ export default async function readmeAPIFetch(
     headers.set('x-readme-source-url', fileOpts.filePath);
   }
 
-  const fullUrl = `${getProxy()}${config.host}${pathname}`;
+  const fullUrl = `${config.host}${pathname}`;
+  const proxy = getProxy();
 
   debug(
-    `making ${(options.method || 'get').toUpperCase()} request to ${fullUrl} with headers: ${sanitizeHeaders(headers)}`,
+    `making ${(options.method || 'get').toUpperCase()} request to ${fullUrl} ${proxy ? `with proxy ${proxy} and ` : ''}with headers: ${sanitizeHeaders(headers)}`,
   );
 
-  return nodeFetch(fullUrl, {
+  return fetch(fullUrl, {
     ...options,
     headers,
+    dispatcher: proxy ? new ProxyAgent(proxy) : undefined,
```

Comment:
lol @ the way node-fetch forced us to handle proxies 🤡

Reply:
lol that was not exactly a node-fetch issue and more of a me issue 😭